### PR TITLE
new feed variant 20220518

### PR DESCRIPTION
**Pull request by dscottS3**
## What type of PR is this? (check all applicable)

- [x] Feature

## Description

New feed variant per Jennie. Thanks @jeremyf!

## Related Tickets & Documents

n/a

## QA Instructions, Screenshots, Recordings

just let the tests pass on this one

### UI accessibility concerns?

n/a

## Added/updated tests?

- [x] No, and this is why: variants are tested directly

## [Forem core team only] How will this change be communicated?

- [x] I will share this change internally with the appropriate teams

### CODE REVIEW

**Comment by jeremyf** [comment]
I ran `diff config/feed-variants/20220518-variant.json config/feed-variants/20220422-variant.json` and got the following.  

```sh
diff config/feed-variants/20220518-variant.json config/feed-variants/20220422-variant.json
8,21c8,21
<         [1, 0.975],
<         [2, 0.965],
<         [3, 0.955],
<         [4, 0.945],
<         [5, 0.935],
<         [6, 0.925],
<         [7, 0.922],
<         [8, 0.919],
<         [9, 0.916],
<         [10, 0.913],
<         [11, 0.91],
<         [12, 0.907],
<         [13, 0.904],
<         [14, 0.901]
---
>         [1, 0.99],
>         [2, 0.985],
>         [3, 0.98],
>         [4, 0.975],
>         [5, 0.97],
>         [6, 0.965],
>         [7, 0.96],
>         [8, 0.955],
>         [9, 0.95],
>         [10, 0.945],
>         [11, 0.94],
>         [12, 0.935],
>         [13, 0.93],
>         [14, 0.925]
51c51
<         [1, 1],
---
>         [0, 0.8],
58c58
<         [1, 1],
---
>         [0, 0.95],
73,74c73,74
<         [1, 0.5],
<         [2, 0.4],
---
>         [1, 0.8],
>         [2, 0.5],
84,85c84,85
<         [2, 0.9],
<         [3, 0.95],
---
>         [2, 0.93],
>         [3, 0.98],
```

I was only expecting the following-ish:

```sh
51c51
<         [1, 1],
---
>         [0, 0.8],
58c58
<         [1, 1],
---
>         [0, 0.95],
```

File: config/feed-variants/20220518-variant.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        [7, 0.94],
+        [8, 0.96],
+        [9, 0.98]
+      ],
+      "fallback": 1
+    },
+    "featured_article": { "cases": [[1, 1]], "fallback": 0.85 },
+    "following_author": {
+      "cases": [
+        [1, 1],
```
Comment (jeremyf) [inline config/feed-variants/20220518-variant.json:51]: This should be `[0, 1],`; Alternatively, you could exclude the `following_author` lever (which would improve query performance.

File: config/feed-variants/20220518-variant.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        [1, 1]
+      ],
+      "fallback": 1
+    },
+    "following_org": {
+      "cases": [
+        [1, 1],
```
Comment (jeremyf) [inline config/feed-variants/20220518-variant.json:58]: This should be `[0, 1],`; alternatively you could exclude the following_org lever to improve query performance. (no sense doing the join work if we're not using it)

**Review by msarit — APPROVED** [review]

File: config/feed-variants/20220518-variant.json (hunk 2 above)
Comment (dscottS3) [reply config/feed-variants/20220518-variant.json:58]: good catch on these. I'm leaning toward keeping them if that's okay to keep things more like the incumbent.

**Pull request merged**